firmware/targets/f7/furi_hal/furi_hal_subghz.c

```diff
@@ -331,8 +337,11 @@ uint32_t furi_hal_subghz_set_frequency(uint32_t value) {
         furi_hal_subghz.regulation = SubGhzRegulationOnlyRx;
     }
 
+    uint32_t corrected_frequency = value * (1000000LL + furi_hal_subghz.calibration) / 1000000;
```

Comment:
this formula is incorrect

Reply:
Could you elaborate, please?

Reply:
Correct. My bad, was testing with another patch applied.